src/main.cpp

```diff
@@ -15,11 +16,15 @@ void handle_input(Window &window) {
 
 void run() {
     Window window("Match Theory", 800, 600);
+    StateManager stateManager = StateManager();
```

Comment:
This should work fine: 
```suggestion
    StateManager stateManager;
```